util/hash_test.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/longhorn/sparse-tools/types"
+	"github.com/stretchr/testify/require"
+)
+
+func randomID(randomIDLenth int) string {
+	id := uuid.New().String()
+	return id[:randomIDLenth]
+}
+
+func TestSetSnapshotHashInfoToChecksumFile(t *testing.T) {
+	assert := require.New(t)
+
+	checksumFileName := randomID(8) + types.DiskChecksumSuffix
```

Comment:
nit: it's better to delete the file in teardown via defer.

Reply:
Addressed. Thank you.